src/QueryBuilder/QueryBuilder/QueryBuilderModal/DataTypeInput/DataTypeInput.js

```diff
@@ -112,9 +114,44 @@ export const DataTypeInput = ({
   };
 
   const stringTypeControls = () => {
+    const { fieldName } = rest;
+    const isPluginOrganizationRequired = fieldName === 'name' || fieldName === 'code';
```

Comment:
This will incorrectly pick up other entity types too and not work with composites containing organizations. Please reach out on teams and we can come up with a better solution for identifying these fields

Reply:
disscussed offline